radix-engine-interface/src/api/actor_api.rs

```diff
@@ -21,4 +22,12 @@ pub trait ClientActorApi<E: Debug> {
     fn actor_get_global_address(&mut self) -> Result<GlobalAddress, E>;
 
     fn actor_get_blueprint(&mut self) -> Result<Blueprint, E>;
+
+    fn actor_call_module_method(
```

Comment:
Not sure if this is necessary. Isn't `actor_call_module_method` = `actor_get_global_address` + `call_method_advanced`?

Slightly worried about the proliferation of APIs. You might argue this saves on API call, but I think there might be other ways for this problem.

Reply:
Yes, me too. Planning on doing an API cleanup pass once we become feature complete. There's a lot that can be cleaned up.